setup.cfg

```diff
@@ -18,7 +18,7 @@ combine_as_imports = True
 addopts = -rxXs
 filterwarnings =
   error
-asyncio_mode = strict
+  ignore: You seem to already have a custom sys.excepthook handler installed. I'll skip installing Trio's custom handler, but this means MultiErrors will not show full tracebacks.:RuntimeWarning
```

Comment:
What's causing this warning, and when can we expect to be able to remove this filter?